Package.swift

```diff
@@ -816,6 +829,57 @@ let package = Package(
       ]
     ),
 
+    .target(
```

Comment:
If not defined specifically, the target will be build for all platforms [defined for Firebase package](https://github.com/firebase/firebase-ios-sdk/blob/c54fad2f541dbbecb3838cfd776ac6ca0b3fc428/Package.swift#L24). If `FirebasePerformance` doesn't support all of them then it either should be wrapped into another target with a conditional dependency like e.g. [here](https://github.com/firebase/firebase-ios-sdk/blob/c54fad2f541dbbecb3838cfd776ac6ca0b3fc428/Package.swift#L557) or the classes should be annotated with availability like e.g. [here](https://github.com/firebase/firebase-ios-sdk/blob/c54fad2f541dbbecb3838cfd776ac6ca0b3fc428/FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h#L440) or [here](https://github.com/firebase/firebase-ios-sdk/blob/c54fad2f541dbbecb3838cfd776ac6ca0b3fc428/FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppAttestProvider.h#L27).

A wrapper target seems to be a more suitable and less time consuming option for FirePerf case.

Reply:
Thanks Maksym! Super helpful! I updated the code as you suggested!